tests/integration/test_model.py

```
@@ -0,0 +1,47 @@
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)